src/sentry/utils/snuba.py

```diff
@@ -350,8 +350,12 @@ def get_project_issues(project_ids, issue_ids=None):
     result = {}
     for h in hashes:
         tombstone_date = tombstones_by_project.get(h.project_id, {}).get(h.hash, None)
-        pair = (h.hash, tombstone_date.strftime("%Y-%m-%d %H:%M:%S") if tombstone_date else None)
-        result.setdefault(h.group_id, []).append(pair)
+        issue = (
+            h.hash,
+            h.project_id,
```

Comment:
I think this is in the wrong place? IIUC the hashes should still be just 2-tuples of `(hash, tombstone)`? 

so the final collection would be something like `issues: [[issue_id, project_id, [[hash, tombstone], [hash2, tombstone2]]]]`

Reply:
Haha, I totally flubbed that one.

Reply:
no worries, that's what reviews are for